tensorflow/core/platform/platform.h

```diff
@@ -45,7 +45,12 @@ limitations under the License.
 
 // Since there's no macro for the Raspberry Pi, assume we're on a mobile
 // platform if we're compiling for the ARM CPU.
-#define IS_MOBILE_PLATFORM
+
+// TODO: Disable this define because it breaks (at least) in:
+//  ./tensorflow/core/platform/gif.h
+//  ./tensorflow/core/platform/jpeg.h
+//  ./tensorflow/core/platform/png.h
+// #define IS_MOBILE_PLATFORM
```

Comment:
heh, my instinct was that we shouldn't do this, but looking at what IS_MOBILE_PLATFORM actually does, there are some 'interesting' decisions there... I expect we don't want this defined, but it'd be nice if we could #undef it rather than altering this file.